shared/models/git_repo.rb

```diff
@@ -223,7 +223,13 @@ def setup_auth(repo_auth: self.repo_auth)
       self.temporary_storage_path = File.join(self.temporary_git_storage, "git-auth-#{SecureRandom.uuid}")
       # More details: https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage
       local_repo_path = self.git_config.local_repo_path
-      FileUtils.mkdir_p(local_repo_path) unless File.directory?(local_repo_path)
+
+      # Creates the `local_repo_path` directory and `notifications/` directory
+      # if they do not exist
+      unless File.directory?(local_repo_path)
+        FileUtils.mkdir_p(local_repo_path)
+        FileUtils.mkdir_p(File.join(local_repo_path, "notifications"))
```

Comment:
I don't know if `notifications` should go here, that directory is totally separate from any `git_repo`, right?

Reply:
Ahh, yep! The `local_repo_path` is `~/fastlane/ci/`. So the notifications would be stored in `~/fastlane/ci/notifications`, separate from the configuration repo. 🙂

Reply:
Oh shoot, I just lied. I want the `@containing_path` 😳